src/main/kotlin/io/github/mojira/arisa/modules/EmptyModule.kt

```diff
@@ -0,0 +1,53 @@
+package io.github.mojira.arisa.modules
+
+import arrow.core.Either
+import arrow.core.extensions.fx
+import arrow.core.left
+import arrow.core.right
+
+data class EmptyModuleRequest(
+    val numAttachments: Int,
+    val description: String?,
+    val environment: String?
+)
+
+const val DESCDEFAULT = "Put the summary of the bug you're having here\r\n\r\n*What I expected to happen was...:*\r\nDescribe what you thought should happen here\r\n\r\n*What actually happened was...:*\r\nDescribe what happened here\r\n\r\n*Steps to Reproduce:*\r\n1. Put a step by step guide on how to trigger the bug here\r\n2. ...\r\n3. ..."
+const val ENVDEFAULT = "Put your operating system (Windows 7, Windows XP, OSX) and Java version if you know it here"
+const val MINLENGTH = 5
+
+class EmptyModule(
+    val resolveAsIncomplete: () -> Either<Throwable, Unit>,
+    val addEmptyComment: () -> Either<Throwable, Unit>
+) : Module<EmptyModuleRequest> {
+    override fun invoke(request: EmptyModuleRequest): Either<ModuleError, ModuleResponse> = Either.fx {
+        with(request) {
```

Comment:
Oh `with` is some neat syntax sugar I wasn't aware of yet! Nice!

Reply:
Kotlin is full of this syntax sugar :D